MIDIchlorians/MIDIchlorians/DataManager.swift

```diff
@@ -195,7 +195,7 @@ class DataManager {
         }
 
         do {
-            try realm?.write { realm?.add(Audio(value: audioFile)) }
+            try realm?.write { realm?.add(Audio(value: [audioFile])) }
```

Comment:
Oh shit my bad
should be value: ["class_property_name": audioFile]

Do you want to merge this in? Or shall I submit all the changes at once in one PR?